src/Parser/BalanceDruid/Modules/Features/AlwaysBeCasting.js

```diff
@@ -31,12 +31,12 @@ class AlwaysBeCasting extends CoreAlwaysBeCasting {
   suggestions(when) {
     const deadTimePercentage = this.totalTimeWasted / this.owner.fightDuration;
     
-    when(deadTimePercentage).isGreaterThan(0.02)
+    when(this.deadTimePercentage).isGreaterThan(0.02)
       .addSuggestion((suggest, actual, recommended) => {
         return suggest(<span> Your downtime can be improved. Try to Always Be Casting (ABC)...</span>)
           .icon('spell_mage_altertime')
-          .actual(`${formatPercentage(this.deadTimePercentage)}% dead GCD time`)
-          .recommended(`${Math.round(formatPercentage(recommended))}% is recommended`)
+          .actual(`${formatPercentage(deadTimePercentage)}% downtime`)
```

Comment:
Using `actual` here is recommended so you don't have to keep this in sync with the `when()` argument.